workspaces/mta/package.json

```diff
@@ -55,7 +55,7 @@
   "resolutions": {
     "@types/react": "^18",
     "@types/react-dom": "^18",
-    "@microsoft/api-extractor": "7.36.4"
+    "@microsoft/api-extractor": "7.47.11"
```

Comment:
According to @awanlin you might be able to remove this resolution now, if you're based on Backstage 1.32.z

Reply:
Yes, that correct. You can remove this line and bump `@backstage/repo-tools` to the latest version.